superduperdb/backends/ibis/query.py

```diff
@@ -815,7 +824,12 @@ def compile(
             return getattr(parent, self.name), tables
         args, tables = _compile_item(self.args, db, tables=tables)
         kwargs, tables = _compile_item(self.kwargs, db, tables=tables)
-        return getattr(parent, self.name)(*args, **kwargs), tables
+        try:
+            return getattr(parent, self.name)(*args, **kwargs), tables
+        except AttributeError:
+            raise QueryValidationError(
```

Comment:
This would be a case for `raise Error from e`

Reply:
getattr(parent, self.name) 

will give attribute error if not found so for this statement `from e` won't make sense

but func(*args, **kwargs) might raise an AttributeError 

I think I need to do 
```python
try:
    query = getattr(parent, self.name)
except AttributeError:
    raise QUeryValidation...
return query(*args, **kwargs), tables

```
@blythed